ZenPacks/zenoss/WBEM/datasources/WBEMDataSource.py

```diff
@@ -258,8 +258,16 @@ def onSuccess(self, results, config):
             ds0.params['result_component_key']
 
         for result in results:
+            result_key_value = None
+
             if result_component_key:
-                datasource = datasources.get(result[result_component_key])
+                result_component_keys = result_component_key.split(',')
```

Comment:
It would be usefull to show some sample data and comments so that later developers can understand what is required here.